test/quantization/test_numeric_suite_fx.py

```diff
@@ -581,8 +581,8 @@ def __init__(self):
                 super().__init__()
 
             def forward(self, x0):
-                x1 = F.relu(x0)
-                y1 = F.relu(x0)
+                x1 = torch.add(x0, 1.0)
```

Comment:
Just curious why change relu to add?

Reply:
the PR changes nodes with equal types to not be matched by default.  Since relu is the same op before and after quantization, it will no longer be matched in this particular test case (since it's not fused to anything).  We change this to an op which does get swapped, just so this test case still makes sense.  Cat is one example, various other ops could have worked too.